src/BookReader/Mode1UpLit.js

```diff
@@ -368,7 +368,7 @@ export class Mode1UpLit extends LitElement {
   computeDefaultScale(page) {
     // Default to real size if it fits, otherwise default to full width
     const containerWidthIn = this.visiblePixelsToWorldUnits(this.htmlDimensionsCacher.clientWidth);
-    return Math.min(1, containerWidthIn / (page.widthInches + 2 * this.SPACING_IN));
+    return Math.min(1, containerWidthIn / (page.widthInches + 2 * this.SPACING_IN)) || 1;
```

Comment:
@cdrini - I found that this can return `0` and make `this.scale = 0` -> this is when I found th theater to be empty.  I added a default value of 1 here (as it Is the starting default) - and everything after that seems to play nicely. lmk what you think, hope this is ok.